config/pipeline.yaml

```diff
@@ -857,6 +857,21 @@ jobs:
         patchlevel: 7
     kcidb_test_suite: kselftest.dt
 
+  kselftest-discoverable-devices-probe:
```

Comment:
We usually use only the name of the test suite at the end of the job name. Maybe better to follow the same naming scheme and rename it to just `kselftest-discoverable-devices` as suite name is `devices`. The same rename should be done for kcidb_test_suite property.

Reply:
Updated the name to match the kselftest collection like the others.